src/steps/index.js

```diff
@@ -1,10 +1,11 @@
-export { setGatsbyApiToState } from "~/steps/set-gatsby-api-to-state"
-export { ensurePluginRequirementsAreMet } from "~/steps/check-plugin-requirements"
-export { ingestRemoteSchema } from "~/steps/ingest-remote-schema"
-export { persistPreviouslyCachedImages } from "~/steps/persist-cached-images"
-export { sourcePreviews } from "~/steps/source-nodes/update-nodes/source-previews"
-export { sourceNodes } from "~/steps/source-nodes"
-export { createSchemaCustomization } from "~/steps/create-schema-customization"
-export { setImageNodeIdCache } from "~/steps/set-image-node-id-cache"
-export { startPollingForContentUpdates } from "~/steps/source-nodes/update-nodes/content-update-interval"
-export { checkIfSchemaHasChanged } from "~/steps/ingest-remote-schema/diff-schemas"
+export { setGatsbyApiToState } from "~/steps/set-gatsby-api-to-state";
```

Comment:
Sorry for the formatting changes :disappointed:

Reply:
There's a .prettierrc file in the root :) if you run prettier using that it will fix it

Reply:
Not sure why my VSCode isn't respecting that config

Reply:
@TylerBarnes I'm not finding the prettier config in this project. If there's actually not one I'd be glad to steal one from wherever, merge that separately, and then rebase this one.